Server/mods/deathmatch/logic/CBlip.cpp

```diff
@@ -91,31 +92,18 @@ bool CBlip::ReadSpecialData(void)
             return false;
         }
     }
-    else
-    {
-        m_Color = SColorRGBA(0, 0, 255, 255);
-    }
 
+    // Grab the "dimension" data
     if (GetCustomDataInt("dimension", iTemp, true))
         m_usDimension = static_cast<unsigned short>(iTemp);
 
     // Grab the "ordering" data
     if (GetCustomDataInt("ordering", iTemp, true))
-    {
-        if (iTemp >= -32768 && iTemp <= 32767)
-        {
-            m_sOrdering = static_cast<short>(iTemp);
-        }
-        else
-        {
-            CLogger::ErrorPrintf("Bad 'ordering' id specified in <blip> (line %u)\n", m_uiLine);

```

Comment:
That's because the final value gets clamped, so there is no bad order anymore.

Reply:
Ok, no problem, I'll add the error back in. I'll also check if we do more clamping elsewhere.

Reply:
Realized that was some old commit that you referred to. Anyway the error is there right now.

So in this PR:
  - Blip `size` gets clamped between 0 and 25. [Function equivalent](https://wiki.multitheftauto.com/wiki/SetBlipSize) clamps too.
  - Blip `visibleDistance` gets clamped between 0 and 65535. [Function equivalent](https://wiki.multitheftauto.com/wiki/SetBlipVisibleDistance) clamps too.

Should I change these two map attributes to result in an error instead rather than clamping silently?

Reply:
Yes, I think reporting it as an error would be good. I think later we can go over those functions and turn them into warnings.

Reply:
Ok, cheers.